### PR TITLE
Update Get-CsOnlineVoiceUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -546,6 +546,16 @@ The following attributes are no longer relevant to Teams and have been dropped f
 - Phone
 - WindowsEmailAddress
 
+The following attributes are temporarily unavailable in the output when using the "-Filter" or when using the cmdlet without any input parameter:
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:549]: Probably better to say - ...... when used without the "-Identity" parameter:

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ Get-CsOnlineDirectoryTenant [[-Tenant] <Guid>] [-DomainController <Fqdn>] [-Forc
 ```
 
 ## DESCRIPTION
-**Note**: Starting with Teams PowerShell Module 4.0 (available from March 1, 2022), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
+**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated on 30th April 2022. Use the Get-CsTenant or  Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]:
```suggestion
**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated. Use the Get-CsTenant or  Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

Comment (JohanFreelancer9) [reply skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]: Microsoft Docs policy is to not publish specific future dates, so I would suggest removing the date.

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -252,7 +252,7 @@ In the Teams PowerShell Module version 3.0.0 or later, filtering functionality i
 1. LineURI set via OnPrem AD.
 2. Direct Routing numbers assigned to users via Set-CsUser.
 
-In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. Now, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
+In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:255]:
```suggestion
In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -263,7 +263,7 @@ In the Teams PowerShell Module version 3.0.0 or later, the format of the Assigne
 **Policy Attributes**:
 
 - PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`
-"Authority" can contain any of these two values: Host or Tenant for a policy type (PreCanned/Default configurations provided by feature team are referred to as Host configurations while Admin-created configurations are considered Tenant configurations)
+"Authority" can contain any of these two values: Host or Tenant for a policy type (Configurations that are provided by Default are referred to as Host configurations while Admin-created configurations are considered Tenant configurations)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:266]:
```suggestion
"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations).
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:549]:
```suggestion
The following attributes are temporarily unavailable in the output when using the "-Filter" or when used without the "-Identity" parameter:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -546,6 +546,16 @@ The following attributes are no longer relevant to Teams and have been dropped f
...
 
+The following attributes are temporarily unavailable in the output when using the "-Filter" or when using the cmdlet without any input parameter:
+- SoftDeletionTimeStamp
+- WhenChanged
+- OnPremHostingProvider
+- OnPremSipEnabled
+- OnPremSipAddress
+- OnPremOptionFlags
+
+Note: The above attributes will be available in the near future
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:557]:
```suggestion
**Note**: These attributes will be available in the near future.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 6]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
 ```
 
 ## DESCRIPTION
-In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for TeamsOnly customers using the ExpandLocation parameter:
+In the Teams PowerShell Module version 3 or later, the following improvements have been introduced for organizations having Teams:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:26]:
```suggestion
In the Teams PowerShell Module version 3 or later, the following improvements have been introduced for organizations using Teams: 
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 7]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
 ```
 
 ## DESCRIPTION
-In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for TeamsOnly customers using the ExpandLocation parameter:
+In the Teams PowerShell Module version 3 or later, the following improvements have been introduced for organizations having Teams: 
+- This cmdlet now accurately returns users that are voice enabled (older cmdlet in 2.6.0 and below used to return users without MCOEV* plans assigned)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:27]:
```suggestion
- This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in 2.6.0 and below returned users without MCOEV* plans assigned).
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 8]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
...
+- Result size is not limited to 100 users anymore (older cmdlet in 2.6.0 and below used to limit result size to 100)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:28]:
```suggestion
- The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and later limited the result size to 100).
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 9]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
...
+
+In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for organizations having Teams users using the ExpandLocation parameter:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:30]:
```suggestion
In the Teams PowerShell Module version 2.6.2 and later, the following attributes have been deprecated for organizations with Teams users using the ExpandLocation parameter:
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 10]
```diff
@@ -32,7 +36,7 @@ In the Teams PowerShell Module version 2.6.2 or later, the following attributes
 - ResultSize
 - LicenceState
 
-In the Teams PowerShell Module version 2.6.2 or later, the following input parameters have been deprecated for TeamsOnly customers due to low or zero usage:
+In the Teams PowerShell Module version 2.6.2 or later, the following input parameters have been deprecated for organizations having Teams users due to low or zero usage:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:39]:
```suggestion
In Teams PowerShell Module version 2.6.2 and later, the following input parameters are deprecated for organizations with Teams users due to low or zero usage:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes if you agree with them. Many thanks.

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: Done

**Comment by isrumnon** [comment]
@JohanFreelancer9 done!

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 11]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
 ```
 
 ## DESCRIPTION
-In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for TeamsOnly customers using the ExpandLocation parameter:
+In Teams PowerShell Module version 3 and later, the following improvements have been introduced for organizations using Teams: 
+- This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in version 2.6.0 and later returned users without MCOEV* plans assigned).
+- The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and later limited the result size to 100).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:28]:
```suggestion
- This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in version 2.6.0 and earlier returned users without MCOEV* plans assigned).
- The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and earlier limited the result size to 100).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @isrumnon, my bad. Please commit the additional suggested change. Many thanks.

**Comment by isrumnon** [comment]
@JohanFreelancer9 done!

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Thank you @isrumnon.

**Comment by yogkumgit** [comment]
@tseward @dariomws @get-itips

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 12]
```diff
@@ -252,7 +252,7 @@ In the Teams PowerShell Module version 3.0.0 or later, filtering functionality i
 1. LineURI set via OnPrem AD.
 2. Direct Routing numbers assigned to users via Set-CsUser.
 
-In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. Now, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
+In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:255]:
```suggestion
In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineUriManuallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 13]
```diff
@@ -23,7 +23,11 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
 ```
 
 ## DESCRIPTION
-In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for TeamsOnly customers using the ExpandLocation parameter:
+In Teams PowerShell Module version 3 and later, the following improvements have been introduced for organizations using Teams:
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:26]:
```suggestion
In Teams PowerShell Module version 3.0 and later, the following improvements have been introduced for organizations using Teams: 
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
ok after @dariomws suggestions

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md [hunk 14]
```diff
@@ -22,7 +22,8 @@ Get-CsOnlineDirectoryTenant [[-Tenant] <Guid>] [-DomainController <Fqdn>] [-Forc
 ```
 
 ## DESCRIPTION
-**Note**: Starting with Teams PowerShell Module 4.0 (available from March 1, 2022), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
+**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated. Use the Get-CsTenant or  Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]: There's an extra space here:

'CsTenant or  Get-Cs'

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md (hunk 14 above)
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]:
```suggestion
**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated. Use the Get-CsTenant or Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

**Comment by isrumnon** [comment]
@yogkumgit Done!

**Review by tseward — APPROVED** [review]
also ok after @dariomws's suggestions

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**